### DISCUSSION TITLE
Cleanup the repository

### DISCUSSION THREAD

**Discussion by stephanbcbauer** [eclipse-tractusx/tutorial-resources#136] [General]
The [repository](https://github.com/eclipse-tractusx/tutorial-resources/tree/main/mxd) contains three files:

![image](https://github.com/eclipse-tractusx/tutorial-resources/assets/84396022/4928a398-a875-432d-b1f8-6f36793e92ae)

The purpose was to describe and use them for the tutorial in another [repository](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/tree/feat/tutorials/docs/tutorials/e2e). Since we already included the information, the files (as separate files) are not needed any more.

The idea would be to delete them, or include the information in the [readme](https://github.com/eclipse-tractusx/tutorial-resources/blob/main/mxd/README.md)

**Comment by stephanbcbauer** [comment]
@jkbquabeck @paullatzelsperger

**Reply by jkbquabeck** [reply]
Since we discussed this already: please go ahead and delete these files, unless @paullatzelsperger has different plans.

**Comment by hemantxpatel** [comment]
It has been done via PR #220
